shape-blog-nextjs/src/pages/posts/[slug].js

```diff
@@ -101,20 +101,24 @@ export async function unstable_getStaticProps({ params }) {
   }
 }
 
-export async function unstable_getStaticPaths() {
+export async function getStaticPaths() {
   try {
     const res = await TakeShape.graphql({query: postSlugsQuery})
     const json = await res.json()
     if (json.errors) throw json.errors
     const data = json.data
     const posts = data.posts.items
-    return posts.reduce((pages, post) => pages.concat({
-      params: {slug: post.slug}
-    }), [])
+
+	const output = {
+		paths: posts.map(post => ({params: {slug: post.slug}})),
+		fallback: false
+	};
+	console.log(output)
```

Comment:
Also remove this one

Reply:
should be done. left these debugs in my accident.